docs/installation/docker-prereq.md

```diff
@@ -0,0 +1,47 @@
+# Prerequisites
+
+## Requirements for using ISLE with Docker Compose
+
+- Docker 19.x+
+- Docker Compose version 1.25.x+
+- GNU Make 4.0+
+- Git 2.0+
+- [ISLE Docker Compose](https://github.com/islandora-devops/isle-dc)
+- At least 8GB of RAM (ideally 16GB)
+- An administrator account
+- (Mac OS) Apple Developer Tools
```

Comment:
Any particular Mac OS version, or warnings about specific versions?

Reply:
Not sure about that. I didn't add this requirements since I'm on linux.  I think this came from @rosiel .  Maybe she can chime in with her mac os version and we can start with that as a minimum.

Reply:
not that I know of. I havent heard of problems with any versions.

Reply:
It has been running without issue on a Macbook Air with 16GB of ram on a m1 processor (11.5Beta). I think I had to recently disable Docker Compose v2 in the settings though.